package.json

```diff
@@ -8,53 +8,40 @@
   "author": "",
   "license": "LGPL-2.1",
   "scripts": {
-    "build": "webpack",
-    "watch": "webpack --watch",
-    "clean": "cd ./dist && rm -rf *",
-    "eslint": "eslint --ext .jsx --ext .js src/ lib/",
-    "eslint:fix": "eslint --fix --ext .jsx --ext .js src/ lib/"
+    "build": "./build.js",
+    "watch": "./build.js -w",
+    "eslint": "eslint --ext .jsx --ext .js src/",
+    "eslint:fix": "eslint --fix --ext .jsx --ext .js src/"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.0",
-    "@babel/eslint-parser": "^7.13.14",
-    "@babel/preset-env": "7.12.17",
-    "@babel/preset-react": "^7.0.0",
     "argparse": "^2.0.1",
-    "babel-loader": "^8.0.0",
-    "chrome-remote-interface": "^0.32.1",
```

Comment:
Hmmmm this is surely still needed, or I guess this was moved to BiDi.. right

Reply:
this was dropped from cockpit package.json in the CDP -> BiDi migration
https://github.com/cockpit-project/cockpit-podman/commit/49598f02074779c4aef8541fec2f31aed579db47#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519